mappings/net/minecraft/world/World.mapping

```diff
@@ -80,30 +318,192 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 6 sizeZ
 	METHOD method_218 isBoxSubmergedInFluid (Lnet/minecraft/class_25;)Z
 		ARG 1 box
+	METHOD method_219 loadEntitySkin (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_220 canMonsterSpawn ()Z
+	METHOD method_221 getRainGradient (F)F
+		ARG 1 delta
 	METHOD method_222 getTopY (II)I
-	METHOD method_223 (IIII)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_223 setBlockMetaWithoutNotifyingNeighbors (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+		ARG 4 meta
+	METHOD method_224 playNoteBlockActionAt (IIIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 soundType
+		ARG 5 pitch
+	METHOD method_225 isFireOrLavaInBox (Lnet/minecraft/class_25;)Z
+		ARG 1 box
+	METHOD method_226 unloadEntitySkin (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_227 tickEntities ()V
+	METHOD method_228 getTopSolidBlockY (II)I
+		ARG 1 x
+		ARG 2 z
 	METHOD method_229 setBlock (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-		ARG 4 id
-	METHOD method_235 (IIII)V
+		ARG 4 blockId
+	METHOD method_230 worldEvent (IIIII)V
+		ARG 1 event
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 data
+	METHOD method_231 remove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_232 doLightingUpdates ()Z
+	METHOD method_233 getSpawnPositionValidityY (II)I
+		ARG 1 x
+		ARG 2 z
+	METHOD method_234 isAir (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_235 blockUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_236 serverRemove (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_237 updateSkyBrightness ()V
+	METHOD method_238 hasChunk (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_239 isPosLoaded (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_240 setBlocksDirty (IIII)V
+		ARG 1 x
+		ARG 2 z
+		ARG 3 minY
+		ARG 4 maxY
+	METHOD method_241 updateEntity (Lnet/minecraft/class_57;)V
+		ARG 1 entity
+	METHOD method_242 tick ()V
+	METHOD method_243 blockUpdateEvent (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_244 notifyNeighbors (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+	METHOD method_245 updateWeatherCycles ()V
+	METHOD method_246 setBlockDirty (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_247 camTransferPowerInDirection (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
+	METHOD method_248 manageChunkUpdatesAndEvents ()V
+	METHOD method_249 canPlaceAt (III)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-	METHOD method_243 (III)V
+	METHOD method_250 isEmittingRedstonePowerInDirection (IIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-	METHOD method_253 (IIII)V
+		ARG 4 direction
+	METHOD method_251 saveDimensionData ()V
+	METHOD method_252 getBrightness (III)I
 		ARG 1 x
-		ARG 2 t
+		ARG 2 y
+		ARG 3 z
+	METHOD method_253 neighborUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
 		ARG 3 z
+		ARG 4 blockId
 	METHOD method_254 getSeed ()J
+	METHOD method_255 getLightLevel (III)I
 	METHOD method_256 getTime ()J
+	METHOD method_257 isTopY (III)Z
```

Comment:
It looks like this is the actual function for determining if a block is exposed to the sky, by finding the highest Y block, or something. Idk. In MCP its called `canExistingBlockSeeTheSky`. There is a function in modern yarn called `isTopSolid` I guess maybe this is similar?

Reply:
will be isTopSolid

Reply:
probably fine as is